src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -33,7 +30,7 @@ class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with
     val blockExecResult = for {
       _ <- validateBlockBeforeExecution(block, blockchain, validators)
 
-      execResult <- executeBlockTransactions(block, blockchain, storages, validators)
+      execResult <- executeBlockTransactions(block, blockchain, storages, validators.signedTransactionValidator)
```

Comment:
Just thinking loud, if we could add `val validators = new Validators(blockchainConfig)` or something like that in `Ledger` we could avoid passing these vals all around